### PR TITLE
split debt to contact into query and handler products

### PR DESCRIPTION
**Pull request by jdmacias-va**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -32,7 +32,8 @@ def products() {
   products["qms"] = ["health_apis_jenkins"]
   products["sfdc-mvi-ent"] = ["health_apis_jenkins"]
   products["sfdc-mpi-msg-receiver"] = ["health_apis_jenkins"]
-  products["sf-dmc-debt-to-contact"] = ["health_apis_jenkins"]
+  products["sf-dmc-debt-to-contact-handler"] = ["health_apis_jenkins"]
+  products["sf-dmc-debt-to-contact-query"] = ["health_apis_jenkins"]
```
Comment (jgarnhart-va) [inline Jenkinsfile:36]: I think these need to match the namespace values. Namely:
sf-d2c-handler
sf-d2c-query

File: ingress.tests [hunk 2]
```diff
@@ -120,11 +120,13 @@ sfdc-mvi-ent /sfdc-mvi-ent/v0/mviRetrieve
...
-# SF DMC Debt to Contact
-sf-dmc-debt-to-contact /sf-dmc-debt-to-contact-query/v0/disableCron
-sf-dmc-debt-to-contact /sf-dmc-debt-to-contact-query/v0/enableCron
-sf-dmc-debt-to-contact /sf-dmc-debt-to-contact-query/v0/putDebtsOnQueueByDate
-sf-dmc-debt-to-contact /sf-dmc-debt-to-contact-query/v0/putDebtsOnQueueByNumber
+# SF DMC Debt to Contact Query
+sf-dmc-debt-to-contact-query /sf-dmc-debt-to-contact-query/v0/disableCron
+sf-dmc-debt-to-contact-query /sf-dmc-debt-to-contact-query/v0/enableCron
+sf-dmc-debt-to-contact-query /sf-dmc-debt-to-contact-query/v0/putDebtsOnQueueByDate
+sf-dmc-debt-to-contact-query /sf-dmc-debt-to-contact-query/v0/putDebtsOnQueueByNumber
```
Comment (jgarnhart-va) [inline ingress.tests:127]: Change to /sf-d2c-query/

File: ingress.tests [hunk 3]
```diff
@@ -120,11 +120,13 @@ sfdc-mvi-ent /sfdc-mvi-ent/v0/mviRetrieve
...
+# SF DMC Debt to Contact Handler
+sf-dmc-debt-to-contact-handler /sf-dmc-debt-to-contact-handler/v0/consumeQueue
```
Comment (jgarnhart-va) [inline ingress.tests:129]: Change to /sf-d2c-handler/

File: products/sf-dmc-debt-to-contact-handler.conf [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# sf dmc debt to contact handler app
+DU_ARTIFACT=health-apis-sfdc-dmc-debt-to-contact-handler-deployment
+DU_VERSION=1.0.49
+DU_NAMESPACE=sf-dtoc-handler
```
Comment (jgarnhart-va) [inline products/sf-dmc-debt-to-contact-handler.conf:4]: What is this namespace?

File: products/sf-dmc-debt-to-contact-handler.conf [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+DU_DECRYPTION_KEY="$DEPLOYMENT_CRYPTO_KEY"
+DU_AUTOMATIC_AVAILABILITY_ZONES=ab
+DU_HEALTH_CHECK_PATH="/sf-dmc-debt-to-contact-handler/v0/actuator/health"
```
Comment (jgarnhart-va) [inline products/sf-dmc-debt-to-contact-handler.conf:7]: /sf-d2c-handler/

File: products/sf-dmc-debt-to-contact-handler.conf [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+DU_PROPERTY_LEVEL_ENCRYPTION=true
+# ========================================
+DU_LOAD_BALANCER_RULES[540]="/sf-dmc-debt-to-contact-handler/v0/*"
```
Comment (jgarnhart-va) [inline products/sf-dmc-debt-to-contact-handler.conf:10]: /sf-d2c-handler/

File: products/sf-dmc-debt-to-contact-query.conf [hunk 7]
```diff
@@ -1,11 +1,10 @@
-# sf dmc debt to contact app
-DU_ARTIFACT=health-apis-sfdc-dmc-debt-to-contact-deployment
-DU_VERSION=1.0.49
-DU_NAMESPACE=sf-dmc-debt-to-contact
+# sf dmc debt to contact query app
+DU_ARTIFACT=health-apis-sfdc-dmc-debt-to-contact-query-deployment
+DU_VERSION=1.0.50
+DU_NAMESPACE=sf-dtoc-query
```
Comment (jgarnhart-va) [inline products/sf-dmc-debt-to-contact-query.conf:4]: What namespace is this?

File: products/sf-dmc-debt-to-contact-query.yaml [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+  name: sf-dmc-debt-to-contact-query-ingress
+  namespace: $DU_NAMESPACE
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+spec:
+  rules:
+    - http:
+        paths:
+          - path: /sf-dmc-debt-to-contact-query/v0/(.*)
```
Comment (jgarnhart-va) [inline products/sf-dmc-debt-to-contact-query.yaml:28]: /sf-d2c-query/

**Comment by bschofield-va** [comment]
I will wait until @mramirez-va or @sbair-va has given a 👍

**Comment by jgarnhart-va** [comment]
@mramirez-va @sbair-va - any chance you can give @bschofield-va the green light?

**Pull request merged**